api/tests/integration/infrastructure/repositories/campaign-participation-overview-repository_test.js

```diff
@@ -14,209 +13,173 @@ describe('Integration | Repository | Campaign Participation Overview', () => {
+          expect(_.map(campaignParticipationOverviews, 'id')).to.exactlyContain([ onGoingParticipation.id, toShareParticipation.id]);
+        });
       });
+    });
+  });
 
-      it('should retrieve the campaign participations of the user with filtered assessment where state is ONGOING and TO_SHARE', async () => {
-        const states = ['ONGOING', 'TO_SHARE'];
-        const rawCampaignParticipationOverviews = await campaignParticipationOverviewRepository.findByUserIdWithFilters({ userId, states });
-        const { campaignParticipationOverviews } = rawCampaignParticipationOverviews;
-
-        expect(campaignParticipationOverviews).to.have.lengthOf(2);
-        expect(_.map(campaignParticipationOverviews, 'campaignTitle')).to.deep.equal(['2 - My campaign', '1 - My campaign']);
-      });
+  context('order', () => {
+    context('when all campaign participation have different status', ()=> {
+      it('orders all campaign participation by status', async () => {
+        const { id: participationArchived } = campaignParticipationOverviewFactory.buildArchived({ userId });
+        const { id: participationEndedId } = campaignParticipationOverviewFactory.buildEnded({ userId });
+        const { id: participationOnGoingId } = campaignParticipationOverviewFactory.buildOnGoing({ userId });
+        const { id: participationToShareId } = campaignParticipationOverviewFactory.buildToShare({ userId });
+        await databaseBuilder.commit();
 
-      it('should retrieve the campaign participations of the user with filtered assessment where state is TO_SHARE and ENDED', async () => {
-        const states = ['TO_SHARE', 'ENDED'];
-        const rawCampaignParticipationOverviews = await campaignParticipationOverviewRepository.findByUserIdWithFilters({ userId, states });
-        const { campaignParticipationOverviews } = rawCampaignParticipationOverviews;
+        const { campaignParticipationOverviews } = await campaignParticipationOverviewRepository.findByUserIdWithFilters({ userId });
+        const campaignParticipationIds = _.map(campaignParticipationOverviews, 'id');
 
-        expect(campaignParticipationOverviews).to.have.lengthOf(2);
-        expect(_.map(campaignParticipationOverviews, 'campaignTitle')).to.deep.equal(['2 - My campaign', '3 - My campaign']);
+        expect(campaignParticipationIds).to.exactlyContainInOrder([participationToShareId, participationOnGoingId, participationEndedId, participationArchived]);
       });
+    });
 
-      it('should retrieve the campaign participations of the user with filtered assessment where state is  ENDED', async () => {
-        const states = ['ENDED'];
-        const rawCampaignParticipationOverviews = await campaignParticipationOverviewRepository.findByUserIdWithFilters({ userId, states });
-        const { campaignParticipationOverviews } = rawCampaignParticipationOverviews;
+    context('when there are campaign participation with the same status', () => {
```

Comment:
`when there are several campaign participations with the status ongoing`

Reply:
En faite l'idée de ce test c'est pas lié au statut `ONGOING`, mais vraiment au faite que les participations est le même statut. On a utilisé `ONGOING`, mais on aurait pu utiliser un autre status ça aurait été vrai aussi. 
Je suis pas chaud pour mettre `ONGOING` dans le `it` je préfère parler de la règle de tri plus génériquement.
Il y a peut-être une amélioration à faire pour qu'on ne pense pas que se soit lié au statut `ONGOING` ?

Reply:
Ok ça me va, je comprends 👍